### ISSUE TITLE
Provide more context when resolving conflicts

### ISSUE THREAD

**Issue by despens** [nextcloud/desktop#5823]
![Screenshot-20230619075904-623x855](https://github.com/nextcloud/desktop/assets/571494/4c5cc25f-12b8-4114-baa1-0b1d1c7eec5e)

The above dialog appears when users choose to "resolve conflicts". The dialog doesn't give enough context to make a reasonable decision
- if the conflicting file has some generic name
- was edited too long ago for the user to remember what that file was or is—maybe the current user didn't edit the file at all

The missing context is:
- path information: where are the conflicting files located?
- editing information: who last edited each of the conflicting files?
- reasonable preview of both versions: users can easily change a file by accident by opening and comparing them, sometimes just setting another last modified date. A neutral preview, perhaps using nextcloud server's file preview rendering capabilities, could help.

This process is quite delicate because users might be put into the position of deciding about a file conflict caused by somebody else when folders are shared within a group. Or they might have no recollection of editing something, perhaps a configuration error, bug, or unknown to them background process changed a file without their knowledge.

**Comment by Lamecarlate** [comment]
I think there is a regression here: some times ago, I could open each of the files in an editor to check their content, within this dialog. There were links to open local or remote versions, as it can be seen of the first image of this page: https://nextcloud.com/blog/nextcloud-desktop-client-3-1-is-here-with-new-conflict-dialog-notification-handling-and-sharing-options/

**Comment by vetash** [comment]
I completely agree with @despens. Based on what the Nextcloud UI offers, it's impossible to draw any conclusions. Furthermore, if you click on the app icon in the tray, it displays the path to all files related to the conflict. Why not show this information in the conflict resolution modal window as well?

It would also be helpful to have a centralized solution for such problems. For example, if an enterprise is synchronizing data exports from a CRM or something similar, it would be nice to see on the server who has issues, so as not to distract the end-users from their work. Even a simple client-side (or server-side) option like "always save the latest version" would be useful. Or, for instance, an option like "Save conflicting files as a copy," meaning replace the original with the file that has the most recent modification date but save the original with a "copy" or "conflict" prefix.
